subsys/bluetooth/host/smp.c

```diff
@@ -1876,6 +1880,10 @@ static void smp_pairing_complete(struct bt_smp *smp, uint8_t status)
 
 		if (bt_auth && bt_auth->pairing_complete) {
 			bt_auth->pairing_complete(conn, bond_flag);
+#if defined(CONFIG_BT_SETTINGS) && defined(CONFIG_BT_GATT_CLIENT)
+			/* Pairing has completed, so persist the CCC and CF data. */
+			k_work_submit(&(smp->work_settings_ccc_and_cf_save));
```

Comment:
minor, but to keep consistency throughout the style used within the subsystem: the () is unnecessary around `smp->work_settings_ccc_and_cf_save`, i.e. the `&` operator will do the right thing also without it